.github/CODEOWNERS

```diff
@@ -9,7 +9,7 @@
 
 .snyk           @centreon/centreon-security
 
-**/src/                 @centreon/centreon-php
+/centreon/src/                 @centreon/centreon-php
```

Comment:
maybe could we try without the directive, as *.php are already affected to centreon-php
```suggestion
```

Reply:
honestly, I'm not sure if this is the cause of the issue, but there are 2 folders with the name "src" in /centreon/src/ and  /centreon/www/front_src/src

Reply:
the second folders is for the frontend

Reply:
You can remove this line.
I tested three combinations and it works fine by removing this deprecated override and using the extensions directly:
https://github.com/centreon/centreon/pull/254
https://github.com/centreon/centreon/pull/255
https://github.com/centreon/centreon/pull/256